File: CazareCamine.Web/ClientApp/src/authentication/models/userModel.ts
export class userModel {
  email: string;
  password: string;
  firstName: string;
  lastName: string;

  constructor(_email: string, _password: string, _firstName: string, _lastName: string) {
    this.email = _email;
    this.password = _password;
    this.lastName = _lastName;
    this.firstName = _firstName;
  }

}

File: CazareCamine.Data/Models/UserModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CazareCamine.Data.Models
{
    public class UserModel
    {
        [Key]
        public int UserId { get; set; }

        [Required]
        public string Email { get; set; }

        [Required]
        public string Password { get; set; }

        [Required]
        [Column(TypeName = "varchar(50)")]
        public string FirstName { get; set; }

        [Required]
        [Column(TypeName = "varchar(50)")]
        public string LastName { get; set; }
    }
}
